sdk/cosmos/azure-cosmos/README.md

```diff
@@ -750,6 +750,52 @@ not being able to recognize the new NonStreamingOrderBy capability that makes ve
 If this happens, you can set the `AZURE_COSMOS_DISABLE_NON_STREAMING_ORDER_BY` environment variable to `"True"` to opt out of this
 functionality and continue operating as usual.*
 
+### Public Preview - Full Text Policy and Full Text Indexes
+We have added new capabilities to utilize full text policies and full text indexing for users to leverage full text search
+utilizing our Cosmos SDK. These two container-level configurations have to be turned on at the account-level
```

Comment:
Link to how to enable them at account-level is missing?

Reply:
based on the way we did the vector search release, we did not share the link there - not sure if it was available and not sure if this one is either